erts/emulator/pcre/local_config.h

```diff
@@ -1,5 +1,7 @@
+/* SPDX-License-Identifier: GPL-3.0-or-later WITH Autoconf-exception-generic-3.0 */
```

Comment:
```suggestion
/* SPDX-License-Identifier: BSD-3-Clause */
```

Reply:
Thanks! I do not know why I thought this was an auto-generated file.
Good catch!